src/app/store/slices/workspaces/workspacesStore.ts

```diff
@@ -97,6 +100,7 @@ const fetchCredentials = createAsyncThunk<void, undefined, { state: RootState }>
       dispatch(workspacesActions.setCredentials(credentials));
       localStorageService.set(STORAGE_KEYS.WORKSPACE_CREDENTIALS, JSON.stringify(credentials));
     }
+    dispatch(workspacesActions.setIsLoadingCredentials(false));
```

Comment:
we have to manage when this fails so that the loader doesn't get stuck in infinity.

Reply:
Done